ddtrace/internal/telemetry/writer.py

```diff
@@ -182,6 +197,9 @@ def app_started_event(self):
     def _app_closing_event(self):
         # type: () -> None
         """Adds a Telemetry request which notifies the agent that an application instance has terminated"""
+        if self._forked:
```

Comment:
This still feels weird to me, we're then defining the "application" as the parent process. This is making an assumption about forking webservers which I don't think holds true in general and so this telemetry event would not be very useful.

Why aren't we just reporting process close/shutdown?

Reply:
resolved offline. Support would prefer telemetry on the application level. The process level can be noisy for applications that fork multiple times. We would also like to keep python telemetry consistent with the other tracers. We can discuss this change for the v2 implementation of the telemetry api